python/lsst/pipe/tasks/getRegionTimeFromVisit.py

```diff
@@ -33,13 +33,10 @@ class GetRegionTimeFromVisitConnections(pipeBase.PipelineTaskConnections,
                                         defaultTemplates={"coaddName": "goodSeeing", "fakesType": ""}):
 
     dummy_visit = pipeBase.connectionTypes.Input(
-        doc="Placeholder connection to provide visit-detector records and "
-            "constrain data IDs to images we're processing. Use of a catalog "
-            "dataset lets us also control how late in the pipeline the task "
-            "is run.",
-        name="{fakesType}{coaddName}Diff_diaSrcTable",
-        storageClass="DataFrame",
-        dimensions={"instrument", "visit", "detector"},
+        doc="Exposure with region and time, I hope!",
```

Comment:
I don't think this doc is an improvement over the original, not least because it's _not_ an exposure, it's a catalog.